lib/compiler-singlepass/src/compiler.rs

```diff
@@ -85,82 +86,109 @@ impl Compiler for SinglepassCompiler {
             .bytes();
         let vmoffsets = VMOffsets::new(pointer_width).with_module_info(&module);
         let import_idxs = 0..module.import_counts.functions as usize;
-        let import_trampolines: PrimaryMap<SectionIndex, _> = import_idxs
-            .into_par_iter_if_rayon()
-            .map(|i| {
-                let i = FunctionIndex::new(i);
-                gen_import_call_trampoline(
-                    &vmoffsets,
-                    i,
-                    &module.signatures[module.functions[i]],
-                    calling_convention,
-                )
-            })
-            .collect::<Vec<_>>()
-            .into_iter()
-            .collect();
+        rayon::ThreadPoolBuilder::new().build().unwrap();
+        rayon::ThreadPoolBuilder::new().build().unwrap();
```

Comment:
These can probably be removed now, right?

Reply:
Yup totally, it's a leftover from my tests, sorry for having missed them when sending the PR!